swagger/operation.go

```diff
@@ -28,12 +38,35 @@ func Interface(s *spec.Swagger, op *spec.Operation) string {
 		input = fmt.Sprintf("i *models.%sInput", capOpID)
 	}
 
-	successType := SuccessType(s, op)
-	if successType == nil {
-		return fmt.Sprintf("%s(ctx context.Context, %s) error", capOpID, input)
+	includeNames := false
+	returnTypes := []string{}
+	returnTypesWithNames := []string{}
```

Comment:
are we getting much value from return types with names? makes things a bit bit messier, and given that i never use them in actual go code i write, i'm curious if think they're worth it. This is something consumers would add to their controller if they really wanted them, right?